### PR TITLE
[A5][Mitigation] Add jwt support

### PR DESCRIPTION
**Pull request by cristiandean**

### CODE REVIEW

File: owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go [hunk 1]
```diff
@@ -13,14 +15,44 @@ func HealthCheck(c echo.Context) error {
 	return c.String(http.StatusOK, "WORKING\n")
 }
 
+func extractClaims(tokenStr string) (jwt.MapClaims, bool) {
+	hmacSecretString := "secret" // Value
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go]: Ops, is that a hardcoded credential on your code? 🧐

**Comment by rafaveira3** [comment]
That's awesome, @cristiandean ! Your strategy of removing userID from the request is pretty good! 👏🏻 I have just made a single comment regarding a hardcoded credential.

File: owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go (hunk 1 above)
Comment (cristiandean) [reply owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go]: Where???? I can't see it =D

<sub><sub><sub>Fixed!</sub></sub></sub>

Comment (rafaveira3) [reply owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go]: 🤣 👏🏻

**Pull request closed without merging**